src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -366,6 +366,29 @@ public void updateGiverEmailOfFeedbackResponseComments(String courseId, String o
     }
     
     /*
+     * Update last editor (normally an instructor email) with the new one
+     */
+    public void updateLastEditorEmailOfFeedbackResponseComments(String courseId, String oldEmail, String updatedEmail) {
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, courseId);
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, oldEmail);
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, updatedEmail);
+        
+        if (oldEmail.equals(updatedEmail)) {
+            return;
+        }
+        
+        List<FeedbackResponseComment> responseComments =
+                this.getFeedbackResponseCommentEntitiesForLastEditorInCourse(courseId, oldEmail);
+        
+        for (FeedbackResponseComment responseComment : responseComments) {
+            responseComment.setLastEditorEmail(updatedEmail);
+        }
+        
+        log.info(Const.SystemParams.COURSE_BACKUP_LOG_MSG + courseId);
```

Comment:
This log doesn't look very useful, we should at least log more information than just the courseId.